corehq/apps/app_manager/util.py

```diff
@@ -359,3 +381,31 @@ def get_commcare_versions(request_user):
     versions = [i.build.version for i in CommCareBuildConfig.fetch().menu
                 if request_user.is_superuser or not i.superuser_only]
     return sorted(versions, key=version_key)
+
+
+def get_usercase_keys(items):
+    n = len(USERCASE_PREFIX)
+    return {k[n:]: v for k, v in items if k.startswith(USERCASE_PREFIX)}
+
+
+def get_usercase_values(items):
+    n = len(USERCASE_PREFIX)
+    return {k: v[n:] for k, v in items if v.startswith(USERCASE_PREFIX)}
+
+
+def any_usercase_items(items):
+    return any(i.startswith(USERCASE_PREFIX) for i in items)
+
+
+def actions_use_usercase(actions):
+    return (any_usercase_items(actions['update_case']['update'].iterkeys()) or
+            any_usercase_items(actions['case_preload']['preload'].itervalues()))
```

Comment:
could make this `action.get('update_case', {}).get('update', {}).iterkeys()` which would make it useful in the SuiteGenerator with `form.active_actions()`